resource_health_check.py

```diff
@@ -112,21 +113,24 @@ def main():
     logger.setLevel('INFO')
     logger.addHandler(log_handler)
 
-    # TODO Figure out how to use SMTPHandler.
-    # server_name = socket.getfqdn()
-    # smtp_handler = SMTPHandler(
-    #     f'{server_name}',
-    #     f'Validator <{getpass.getuser()}@{server_name}>',
-    #     'mrakitin@bnl.gov', f'Health check from {server_name}')
-    # smtp_handler.setFormatter(LogFormatter())
-    # smtp_handler.setLevel('WARNING')
-    # logger.addHandler(smtp_handler)
-
     for email_address in (args.emails or []):
-        mail_handler = LinuxMailHandler(email=email_address)
-        mail_handler.setFormatter(LogFormatter())
-        mail_handler.setLevel('WARNING')
-        logger.addHandler(mail_handler)
+        server_name = socket.gethostname()
+        smtp_handler = SMTPHandler(
+            mailhost='localhost',
+            fromaddr=f'Resource Health Check <noreply@{server_name}>',
+            toaddrs=email_address,
```

Comment:
This argument accepts one email address _or_ a list of email addresses, confirmed in the CPython source here:

https://github.com/python/cpython/blob/9f3fc6c5b4993f2b362263b494f84793a21aa073/Lib/logging/handlers.py#L997-L998

Given that, we can create (at most) one `SMTPHandler` instance and reduce the overhead imposed by the logging framework.

Reply:
Done.